2024-03-31 22:58:27,649 INFO    Thread-17 (_run_job):21556 [wandb_setup.py:_flush():76] Current SDK version is 0.16.5
2024-03-31 22:58:27,650 INFO    Thread-17 (_run_job):21556 [wandb_setup.py:_flush():76] Configure stats pid to 21556
2024-03-31 22:58:27,650 INFO    Thread-17 (_run_job):21556 [wandb_setup.py:_flush():76] Loading settings from C:\Users\jinya\.config\wandb\settings
2024-03-31 22:58:27,650 INFO    Thread-17 (_run_job):21556 [wandb_setup.py:_flush():76] Loading settings from D:\MScDataSparqProject\Supporting_files\wandb\settings
2024-03-31 22:58:27,650 INFO    Thread-17 (_run_job):21556 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'project': 'datasparq', 'entity': 'datasparq', 'root_dir': 'D:\\MScDataSparqProject\\Supporting_files', 'sweep_id': '5nri6c6a', 'run_id': 'y7g60n4q', 'sweep_param_path': 'D:\\MScDataSparqProject\\Supporting_files\\wandb\\sweep-5nri6c6a\\config-y7g60n4q.yaml'}
2024-03-31 22:58:27,650 INFO    Thread-17 (_run_job):21556 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-31 22:58:27,650 INFO    Thread-17 (_run_job):21556 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'Supporting_files\\main_test.py', 'program_abspath': 'd:\\MScDataSparqProject\\Supporting_files\\main_test.py', 'program': 'd:\\MScDataSparqProject\\Supporting_files\\main_test.py'}
2024-03-31 22:58:27,650 INFO    Thread-17 (_run_job):21556 [wandb_init.py:_log_setup():527] Logging user logs to D:\MScDataSparqProject\Supporting_files\wandb\run-20240331_225827-y7g60n4q\logs\debug.log
2024-03-31 22:58:27,650 INFO    Thread-17 (_run_job):21556 [wandb_init.py:_log_setup():528] Logging internal logs to D:\MScDataSparqProject\Supporting_files\wandb\run-20240331_225827-y7g60n4q\logs\debug-internal.log
2024-03-31 22:58:27,651 INFO    Thread-17 (_run_job):21556 [wandb_init.py:init():567] calling init triggers
2024-03-31 22:58:27,651 INFO    Thread-17 (_run_job):21556 [wandb_init.py:init():574] wandb.init called with sweep_config: {'batch_size': 16, 'discount': 0.12443392358894814, 'epochs': 20, 'epsilon': 0.1360806408292181, 'epsilon_state_exploration': 0.8, 'learning_rate': 0.05630856489411432, 'num_episodes': 5, 'num_sample': 50, 'planning_steps': 10, 'target_update_frequency': 2, 'tau': 0.00013002675987952014, 'time_steps': 10, 'w1': 0.5, 'w2': 0.5}
config: {}
2024-03-31 22:58:27,651 INFO    Thread-17 (_run_job):21556 [wandb_init.py:init():612] wandb.init() called when a run is still active
2024-03-31 22:58:28,910 ERROR   Thread-17 (_run_job):21556 [wandb_init.py:init():1215] error
Traceback (most recent call last):
  File "d:\py\Lib\site-packages\wandb\sdk\wandb_init.py", line 1187, in init
    run = wi.init()
          ^^^^^^^^^
  File "d:\py\Lib\site-packages\wandb\sdk\wandb_init.py", line 613, in init
    with telemetry.context() as tel:
  File "d:\py\Lib\site-packages\wandb\sdk\lib\telemetry.py", line 42, in __exit__
    self._run._telemetry_callback(self._obj)
  File "d:\py\Lib\site-packages\wandb\sdk\wandb_run.py", line 763, in _telemetry_callback
    self._telemetry_flush()
  File "d:\py\Lib\site-packages\wandb\sdk\wandb_run.py", line 774, in _telemetry_flush
    self._backend.interface._publish_telemetry(self._telemetry_obj)
  File "d:\py\Lib\site-packages\wandb\sdk\interface\interface_shared.py", line 101, in _publish_telemetry
    self._publish(rec)
  File "d:\py\Lib\site-packages\wandb\sdk\interface\interface_sock.py", line 51, in _publish
    self._sock_client.send_record_publish(record)
  File "d:\py\Lib\site-packages\wandb\sdk\lib\sock_client.py", line 221, in send_record_publish
    self.send_server_request(server_req)
  File "d:\py\Lib\site-packages\wandb\sdk\lib\sock_client.py", line 155, in send_server_request
    self._send_message(msg)
  File "d:\py\Lib\site-packages\wandb\sdk\lib\sock_client.py", line 152, in _send_message
    self._sendall_with_error_handle(header + data)
  File "d:\py\Lib\site-packages\wandb\sdk\lib\sock_client.py", line 130, in _sendall_with_error_handle
    sent = self._sock.send(data)
           ^^^^^^^^^^^^^^^^^^^^^
ConnectionResetError: [WinError 10054] 远程主机强迫关闭了一个现有的连接。
